source/include/ota.h

```diff
@@ -405,13 +405,19 @@ OtaErr_t OTA_Resume( void );
 /* @[declare_ota_resume] */
 
 /**
- * @brief OTA agent task function.
+ * @brief OTA agent event processing loop.
+ *
+ * This is the main event loop and the entry point to the OTA Task.
```

Comment:
This is the main event loop to handle events for OTA update and needs to be called by the application task.